src/com/untamedears/realisticbiomes/RealisticBiomes.java

```diff
@@ -332,15 +342,14 @@ private void registerEvents() {
 			pm.registerEvents(new GrowListener(this), this);
 			pm.registerEvents(new SpawnListener(materialGrowth, fishSpawn), this);
 			pm.registerEvents(new PlayerListener(this, materialGrowth), this);
+			pm.registerEvents(new ClickableInventoryListener(), this);

```

Comment:
I just meant the javadoc. 

The problem you are describing will only occur if you are running a server without any plugins extending ACivMod, but I just realized thats something that should be fixed in CivModCore. I'll do that right now, after that you wont have to register the listener here (and shouldnt because it will break things if it is registered twice).